sympy/printing/tests/test_str.py

```diff
@@ -1166,3 +1166,7 @@ def test_printing_str_array_expressions():
     M = MatrixSymbol("M", 3, 3)
     N = MatrixSymbol("N", 3, 3)
     assert sstr(ArrayElement(M*N, [x, 0])) == "(M*N)[x, 0]"
+
+def test_issue_19435():
+    assert str(1/UnevaluatedExpr(sqrt(pi))) == '1/(sqrt(pi))'
+    
```

Comment:
"No newline at end of file" means: don't put this newline at the end of the file

Reply:
I know what it means. I've done it, and when I ran the linter, it stated that I should add this newline.

Reply:
Can you show what you got? Because this newline should be removed before this PR can be merged